.github/workflows/auto-release.yml

```diff
@@ -25,7 +25,6 @@ jobs:
       - name: Tool Setup
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7.10
```

Comment:
why did we remove the python version

Reply:
That version of python was no longer available in the system, also I didn't find any dependency on the python version while testing.